chia/cmds/coin_funcs.py

```diff
@@ -36,7 +36,7 @@ async def async_list(args: Dict[str, Any], wallet_client: WalletRpcClient, finge
         return
     final_min_coin_amount: uint64 = uint64(int(min_coin_amount * mojo_per_unit))
     final_max_coin_amount: uint64 = uint64(int(max_coin_amount * mojo_per_unit))
-    final_excluded_amounts: List[uint64] = [uint64(int(amount * mojo_per_unit)) for amount in excluded_amounts]
+    final_excluded_amounts: List[uint64] = [uint64(int(Decimal(amount) * mojo_per_unit)) for amount in excluded_amounts]
```

Comment:
I don't know what we usually do, but in general I would rather this happen right at the CLI.  But, why only this one and not the other amounts?

Reply:
The others happen a few lines up at the top of the function:
```
    min_coin_amount = Decimal(args["min_coin_amount"])
    max_coin_amount = Decimal(args["max_coin_amount"])
```

Reply:
I see, they are handled further up.  Probably good to do them all in one place.  `Decimal` conversion above and mojo conversion here.